frontend/packages/console-shared/src/components/dashboard/status-card/AlertItem.tsx

```diff
@@ -28,7 +28,7 @@ export const StatusItem: React.FC<StatusItemProps> = ({ Icon, timestamp, message
   return (
     <div className="co-status-card__alert-item">
       <div className="co-status-card__alert-item-icon co-dashboard-icon">
-        <Icon />
+        <Icon alt="Status alert" title="Status alert" />
```

Comment:
We need to put a real status in the alt here for whatever icon is passed. I believe you can move this into `getSeverityIcon` above.

```tsx
const CriticalIcon = <RedExclamationCircleIcon alt="Critical" title="Critical" />;
const WarningIcon = <YellowExclamationTriangleIcon alt="Warning" title="Warning" />;
const getSeverityIcon = (severity: string) => {
  switch (severity) {
    case 'critical':
      return CriticalIcon;
    case 'warning':
    default:
      return WarningIcon;
  }
};
```